helm_deploy/values-preprod.yaml

```diff
@@ -1,8 +1,13 @@
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
 generic-service:
+
```

Comment:
Should this yaml file also be defining the replicaCount ?
Obv this is nothing to do with your change

Reply:
As discussed, default in generic chart is 4 but we have agreed that for a service getting so little use, 2 should be fine.